src/hostedPages/layouts/HostedPageLayout/HostedPageLayout.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react'
+import { Navbar } from '../../../molecules'
+import { CloseButton } from './CloseButton'
+import classes from './hostedPageLayout.module.scss'
+
+export interface HostedPageLayoutProps {
+  children: React.ReactNode | string
+  onCloseHostedPage: () => void
+  logo?: string
+}
+
+export const HostedPageLayout: FC<HostedPageLayoutProps> = ({
+  children,
+  onCloseHostedPage,
+  logo,
+}) => {
+  return (
+    <div className={classes.layout_container}>
+      <CloseButton onClose={onCloseHostedPage} />
+      <header>
+        <Navbar logo={logo} />
```

Comment:
How is [AST-4525](https://awellhealth.atlassian.net/browse/AST-4525) handled in this div ?

Can we show a loader here when logo string is `null` here?

Reply:
Loading state is handled in Hosted pages repo, see https://github.com/awell-health/hosted-pages/blob/c7b3a0863709def0d5c461cec48458d66682f4de/pages/index.tsx

I decided to just show white page until session data is loaded instead of another loader.